src/main/resources/db.scripts.mvp/delete_migrated_visits_more_than_6_months_in_future.sql

```diff
@@ -0,0 +1,27 @@
+BEGIN;
+
```

Comment:
This work has already been done, 

R_remove_legacy_visits_that_are_six_months_in_future.sql

Why not just use that.

Reply:
Or better - prevent them from being migrated in the first place - return a error if visit dates are too far in the future.

Reply:
Other Script has been run